add Dutch translation for 2.0 (#877)
ContributorCovenant/contributor_covenant: content/version/2/0/code_of_conduct.nl.md

+++
version = "2.0"
aliases = ["/version/2/0"]
+++

# Contributor Convenant Gedragscode (Code of Conduct)

## Onze belofte

> Wij als leden, deelnemers en leiders beloven om deelname aan onze community een intimidatievrije ervaring te maken voor iedereen, ongeacht leeftijd, lichaamsbouw, zichtbare of onzichtbare handicap, etniciteit, geslachtskenmerken, genderidentiteit en expressie, ervaringsniveau, opleiding, sociaaleconomische status, nationaliteit, persoonlijk voorkomen, ras, religie of seksuele identiteit en oriëntatie.

Reviewer: I think that "deelnemers" is not quite right as a translation for "contributors". I would favor something else like "inzenders".

Author: @brodybits what about "bijdragers", like in 1.4? https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/1/4/code-of-conduct.nl.md
Also I now notice a trailing "n" in Contributor Covenant.

Reviewer 2: "deelnemers" feels like an appropriate translation to me

Reviewer 3: Contribuanten would be the correct wording

Reviewer: "bijdragers" sounds fine for me ("deelnemers" made me think of "shareholders" or "participants")

"contribuanten" (probably should add another "t" --> "contributanten") sounds more clear to me, though I don't know how proper it is (I left Amsterdam almost 4 years ago)

Reviewer 3: leiders should be consistent. Further down it's called Gemeenschapsleiders - perhaps thats a better definition

Reviewer 3: Contributant (with extra t) is French, not dutch - in dutch it would be contribuanten https://www.encyclo.nl/begrip/contribuant

Reviewer 3: In general: I'm suggesting using the word community. It's in the woordenlijst from the Taalunie - http://woordenlijst.org/#/?q=community and is well-recognized by Dutch and Flemish people (91%,86% in 2013). So I think we're good there.

> Wij als leden, deelnemers en leiders beloven om deelname aan onze community een intimidatievrije ervaring te maken voor iedereen, ongeacht leeftijd, lichaamsbouw, zichtbare of onzichtbare handicap, etniciteit, geslachtskenmerken, genderidentiteit en expressie, ervaringsniveau, opleiding, sociaaleconomische status, nationaliteit, persoonlijk voorkomen, ras, religie of seksuele identiteit en oriëntatie.

Reviewer 3: ```suggestion
Wij als leden, contribuanten en communityleiders beloven om deelname aan onze community een intimidatievrije ervaring te maken voor iedereen, ongeacht leeftijd, lichaamsbouw, zichtbare of onzichtbare handicap, etniciteit, geslachtskenmerken, genderidentiteit en -expressie, ervaringsniveau, opleiding, sociaal-economische status, nationaliteit, voorkomen, ras, religie of seksuele identiteit en -oriëntatie.
```

> We beloven om te handelen en te communiceren op een manier die bijdraagt aan een open, gastvrije, diverse, inclusieve en gezonde gemeenschap.

Reviewer 3: ```suggestion
We beloven om te handelen en te communiceren op een manier die bijdraagt aan een open, gastvrije, diverse, inclusieve en gezonde community.
```

## Onze standaarden

> Voorbeelden van gedrag dat bijdraagt ​​aan een positieve omgeving voor onze community omvatten:

Reviewer 3: ```suggestion
Voorbeelden van gedrag dat bijdraagt ​​aan een positieve omgeving voor onze community:
```

* Empathie en vriendelijkheid tonen tegenover andere mensen
* Verschillende meningen, standpunten en ervaringen respecteren
* Constructieve feedback geven en gracieus accepteren
* Verantwoordelijkheid aanvaarden en excuses aanbieden aan degenen die door onze fouten zijn getroffen,
  en leren van de ervaring
* Focussen op wat het beste is voor de algemene gemeenschap, niet alleen voor ons als individuen

Voorbeelden van onaanvaardbaar gedrag zijn:

> * Het gebruik van geseksualiseerde taal of beelden, en seksuele aandacht of voorschotten van welke aard dan ook

Reviewer 3: ```suggestion
* Het gebruik van geseksualiseerde taal of beelden en seksuele aandacht of toenadering van welke aard dan ook
```

> * "Trollen", beledigende of denigrerende opmerkingen, en persoonlijke of politieke aanvallen

Reviewer 3: ```suggestion
* "Trollen", beledigende of denigrerende opmerkingen en persoonlijke- of politieke aanvallen
```

> * Openbare of privé-intimidatie

Reviewer 3: ```suggestion
* Openbare- of privé-intimidatie
```

* Publiceren van persoonlijke informatie van anderen, zoals een fysiek adres of e-mail adres, zonder hun uitdrukkelijke toestemming
* Ander gedrag dat redelijkerwijs als ongepast zou kunnen worden beschouwd in een professionele setting

## Handhaving

> Gemeenschapsleiders zijn verantwoordelijk voor het verduidelijken en handhaven van onze normen van acceptabel gedrag en zal passende en eerlijke corrigerende maatregelen nemen in reactie op elk gedrag dat zij ongepast, bedreigend, aanstootgevend, of schadelijk vinden.

Reviewer 4: ... ~zal~ **zullen** passende en eerlijke ...

> Gemeenschapsleiders zijn verantwoordelijk voor het verduidelijken en handhaven van onze normen van acceptabel gedrag en zal passende en eerlijke corrigerende maatregelen nemen in reactie op elk gedrag dat zij ongepast, bedreigend, aanstootgevend, of schadelijk vinden.

Reviewer 3: ```suggestion
Communityleiders zijn verantwoordelijk voor het verduidelijken en handhaven van onze normen van acceptabel gedrag en zullen passende en eerlijke corrigerende maatregelen nemen in reactie op elk gedrag dat zij ongepast, bedreigend, aanstootgevend, of schadelijk vinden.
```

> Gemeenschapsleiders hebben het recht en de verantwoordelijkheid om opmerkingen, commits, code, wiki-bewerkingen, issues en andere bijdragen die niet in overeenstemming met deze Gedragscode zijn te verwijderen, bewerken of afwijzen, en zal redenen voor moderatie meedelen waar nodig.

Reviewer 4: ... ~zal~ **zullen** redenen voor ...

> Gemeenschapsleiders hebben het recht en de verantwoordelijkheid om opmerkingen, commits, code, wiki-bewerkingen, issues en andere bijdragen die niet in overeenstemming met deze Gedragscode zijn te verwijderen, bewerken of afwijzen, en zal redenen voor moderatie meedelen waar nodig.

Reviewer 3: ```suggestion
Communityleiders hebben het recht en de verantwoordelijkheid om opmerkingen, commits, code, wiki-bewerkingen, issues en andere bijdragen die niet in overeenstemming met deze Gedragscode zijn te verwijderen, bewerken of afwijzen en zullen redenen voor moderatie meedelen waar nodig.
```

## Toepassingsgebied

> Deze gedragscode is van toepassing binnen alle gemeenschappelijke ruimtes, en is ook van toepassing wanneer een individu officieel de gemeenschap in openbare ruimtes vertegenwoordigt.

Reviewer 3: ```suggestion
Deze Gedragscode is van toepassing binnen alle gemeenschappelijke ruimtes en is ook van toepassing wanneer een individu officieel de community in openbare ruimtes vertegenwoordigt.
```

> Voorbeelden van het vertegenwoordigen van onze gemeenschap zijn onder meer het gebruik van een officieel e-mailadres, posten via een officieel social media-account, of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.

Reviewer 4: ... ~posten~ **berichten plaatsen** via een officieel  ...

> Voorbeelden van het vertegenwoordigen van onze gemeenschap zijn onder meer het gebruik van een officieel e-mailadres, posten via een officieel social media-account, of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.

Reviewer 3: ```suggestion
Voorbeelden van het vertegenwoordigen van onze community zijn onder meer het gebruik van een officieel e-mailadres, berichten plaatsen via een officieel social media account of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.
```

## Handhaving

Beledigend, intimiderend of anderszins onaanvaardbaar gedrag kan gerapporteerd worden aan de gemeenschapsleiders die verantwoordelijk zijn voor handhaving via [CONTACTMETHODE INVOEGEN].
Alle klachten zullen onmiddellijk en eerlijk worden beoordeeld en onderzocht.

> Alle gemeenschapsleiders zijn verplicht om de privacy en veiligheid van de verslaggever van een incident te respecteren.

Reviewer 3: ```suggestion
Alle communityleiders zijn verplicht om de privacy en veiligheid van de melder van een incident te respecteren.
```

Reviewer: Should this kind of change apply to all instances of "gemeenschap"? (I still see some instances of "gemeenscap" including "gemeenscapleiders" with no suggested changes.)

Reviewer 3: Yes I think I've catched all instances - but they're still in "pending"

Reviewer 3: Allrighty, hope it's now more consistent

## Handhavingsrichtlijnen

> Gemeenschapsleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:

Reviewer 4: I suggest to always use **Gedragscode** with a capital G. The EN translation also uses capitals.

> Gemeenschapsleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:

Reviewer 3: ```suggestion
Communityleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:
```

### 1. Correctie

> **Impact op de gemeenschap**: gebruik van ongepast taalgebruik of ander gedrag dat onprofessioneel wordt geacht, of niet welkom in de gemeenschap.

Reviewer 4: ... niet welkom **is** in de ....

or

.. niet wordt geaccepteerd in de ...

Reviewer 5: I would go with the second option from Marc

Reviewer: Just wondering if there could be any way to simplify the Dutch translation?

> **Impact op de gemeenschap**: gebruik van ongepast taalgebruik of ander gedrag dat onprofessioneel wordt geacht, of niet welkom in de gemeenschap.

Reviewer 3: ```suggestion
**Impact op de community**: Gebruik van ongepast taalgebruik en ander gedrag dat onprofessioneel of als niet gewenst wordt geacht  in de community.
```

> **Gevolg**: een persoonlijke, schriftelijke waarschuwing van community leiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.

Reviewer 4: ... ~community leiders~ **gemeenschapsleiders**  ..

... **met** verduidelijking ...

> **Gevolg**: een persoonlijke, schriftelijke waarschuwing van community leiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.

Reviewer 3: ```suggestion
**Consequentie**: Een persoonlijke, schriftelijke waarschuwing van communityleiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt hierbij om een ​​openbare verontschuldiging gevraagd.
```

> **Gevolg**: een persoonlijke, schriftelijke waarschuwing van community leiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.

Reviewer 3: ```suggestion
**Consequentie**: een persoonlijke, schriftelijke waarschuwing van communityleiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.
```

### 2. Waarschuwing

> **Impact op de gemeenschap**: een overtreding door een enkel incident of reeks van acties.

Reviewer 3: ```suggestion
**Impact op de community**: Een overtreding door een enkel incident of reeks van acties.
```

> **Consequentie**: een waarschuwing met gevolgen als het gedrag voortduurt. Geen interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de gedragscode handhaven, voor een bepaalde periode. Dit omvat het vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals op social media. Het overtreden van deze voorwaarden kan leiden tot een tijdelijke of permanente ban.

Reviewer 4: ~Consequentie~ **Gevolg**

Reviewer 3: Consequentie is a beautiful word I think

Reviewer 4: Yes, it is a beautiful word! But all the other sections describe **Gevolg** 😅. Or change everything to **Consequentie**.

Reviewer: As an international foreigner I would be biased to "change everything to **Consequentie**".

> **Consequentie**: een waarschuwing met gevolgen als het gedrag voortduurt. Geen interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de gedragscode handhaven, voor een bepaalde periode. Dit omvat het vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals op social media. Het overtreden van deze voorwaarden kan leiden tot een tijdelijke of permanente ban.

Reviewer 3: ```suggestion
**Consequentie**: Een waarschuwing met gevolgen als het gedrag voortduurt. Geen interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de Gedragscode handhaven, voor een bepaalde periode. Dit omvat het vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals op social media. Het overtreden van deze voorwaarden kan leiden tot een tijdelijke of permanente ban.
```

Reviewer: I think that **blokkade** would mean blocking and lose consistency with the "permanent ban" in the English version. Also there are several more uses of "ban" in this proposed translation ... including the "Permanente ban" header below

Reviewer 3: Well I just checked, and bannen is in the dictionary nowadays with an expected definition, you learn everyday 😅

### 3. Tijdelijk verbod

> **Impact op de gemeenschap**: een ernstige schending van gemeenschapsnormen, inclusief aanhoudend ongepast gedrag.

Reviewer 3: ```suggestion
**Impact op de community**: Een ernstige schending van community normen, inclusief aanhoudend ongepast gedrag.
```

> **Gevolg**: een tijdelijk verbod op elke vorm van interactie of openbare

Reviewer 3: ```suggestion
**Consequentie**: Een tijdelijk verbod op elke vorm van interactie of openbare
```

> communicatie met de gemeenschap gedurende een bepaalde periode. Geen publieke of privé-interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de Gedragscode handhaven, is toegestaan ​​tijdens deze periode.

Reviewer 4: This consequence describes 'No _public and private_ interactions' while the consequence under II only describes 'No interactions'. It also does in the EN translation I see. Perhaps both descriptions need to be explicit and mention 'public and private interactions'?

> communicatie met de gemeenschap gedurende een bepaalde periode. Geen publieke of privé-interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de Gedragscode handhaven, is toegestaan ​​tijdens deze periode.

Reviewer 3: ```suggestion
communicatie met de community gedurende een bepaalde periode. Geen publieke- of privé-interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de Gedragscode handhaven, is toegestaan ​​tijdens deze periode.
```

> Het overtreden van deze voorwaarden kan leiden tot een permanente ban.

Reviewer 3: ```suggestion
Het overtreden van deze voorwaarden kan leiden tot een permanente ban.
```

Reviewer: Same as my other comment regarding **blokkade** vs **ban**

Reviewer 3: Fixed

### 4. Permanente ban

> **Impact op de gemeenschap**: een patroon van schending van de gemeenschapsstandaarden vertonen, waaronder aanhoudend ongepast gedrag, intimidatie van een individu, of agressie jegens of kleinering van groepen van individuen.

Reviewer 3: ```suggestion
**Impact op de community**: Een patroon van schending van de community standaarden vertonen, waaronder aanhoudend ongepast gedrag, intimidatie van een individu of agressie jegens- of kleinering van groepen van individuen.
```

> **Gevolg**: een permanent verbod op elke vorm van openbare interactie in de gemeenschap.

Reviewer 3: ```suggestion
**Consequentie**: Een permanent verbod op elke vorm van openbare interactie in de community.
```

> **Gevolg**: een permanent verbod op elke vorm van openbare interactie in de gemeenschap.

Reviewer 4: I'm getting confused about the public and private interactions 😅. Here it only describes public interactions with the community (as it does in the EN translation). It does not mention public/private interactions with people involved. Is this intentional?

Has nothing to do with the translation though, so feel free to ignore it here! I can post an issue or make a PR for the EN version once I know if the difference is intentional or not.

Reviewer 3: I'd assume it'd be public & private given the severity - but a great discussion for the EN one indeed

## Attributie

Deze gedragscode is een aangepaste versie van de [Contributor Covenant][homepage], versie 2.0, beschikbaar op [https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].

> Richtlijnen voor impact op de gemeenschap zijn geïnspireerd door [Mozilla's code of conduct enforcement ladder][Mozilla CoC].

Reviewer 3: ```suggestion
Richtlijnen voor impact op de community zijn geïnspireerd door [Mozilla's code of conduct enforcement ladder][Mozilla CoC].
```
